Bio/pairwise2.py

```diff
@@ -83,7 +83,7 @@
     >>> from Bio.pairwise2 import format_alignment
     >>> print(format_alignment(*alignments[0]))
     ACCGT
-    | || 
+    | ||
```

Comment:
Your editor has probably removed these important trailing white spaces, and thus broken the example :(